licenserc.toml

```diff
@@ -20,6 +20,7 @@ excludes = [
     "**/src/test/**/*.txt",
     "**/src/test/**/*.xml",
 
+    "hawkeye-maven-plugin/src/test/java/io/korandoru/hawkeye/package-info.java",
```

Comment:
Maybe I SHOULD not use the Java package-info file as a test file. Do you mind it?

Reply:
Why? If you want to use against Java, you can put one outside the sources tree.